src/sc_generation/abi_generation/mod.rs

```diff
@@ -1,3 +1,4 @@
+/*
```

Comment:
should this file be removed?

Reply:
I'm think we can keep it for now (in case we need to update the previous runtime for whatever reason), but maybe we can 😁